docs/api/security/psa_protected_storage.md

```diff
@@ -0,0 +1,19 @@
+## PSA Protected Storage
+
+### Description
+
+PSA Protected storage APIs allows saving and retrieving data from PSA protected storage.
+
+Unlike [PSA Internal storage](./psa_internal_storage.md), PSA Protected storage implemented to always run in NSPE side and redirect the calls to KVStore instance.
```

Comment:
NSPE side and redirect the calls to KVStore ~~instance~~ static API.

Reply:
In general, the recommended storage API to be used in NSPE is the KVSTore static API. See ../storage/KVStoreGlobalAPI.md. The protected storage API is presented for compatibility with PSA standard